src/pybind/mgr/mgr_module.py

```diff
@@ -304,6 +311,20 @@ def __del__(self):
     def log(self):
         return self._logger
 
+    def cluster_log(self, channel, priority, record):
+        """
+        :param channel: The log channel. This can be 'cluster', 'audit', ...
+        :type channel: string
```

Comment:
`s/string/str/` to make it work with static type checking.

Reply:
Done